### DISCUSSION TITLE
how to get copilot voice access？

### DISCUSSION THREAD

**Discussion by alanhe421** [community/community#102200] [Copilot Conversations] [answered]
I had already signed up for the waitlist when the official announcement about the voice feature was made, for at least over half a year now. I hope to be granted the opportunity to experience it. I have always been a loyal subscriber to Copilot. I now need to use the voice feature to facilitate my child's development and use at home.

GitHub discord can't help me.

![image](https://github.com/community/community/assets/9245110/ad8f0044-3125-4a34-88a9-e8648cbd2841)

**Comment by Akash1134** [comment]
Hi @alanhg ,

Please [sign up for the technical preview](https://githubnext.com/projects/copilot-voice/) and you will receive further information on registered email. 

Thanks!

**Reply by alanhe421** [reply]
I have applied from the beginning, but have not received any notification yet.

**Reply by AliAkhgar** [reply]
me too

**Reply by DianaMToledo** [reply]
Did you manage to get it? I also want access....(I have applied already)

**Comment by unowiz** [comment]
Any updates on voice access? I've installed the extension but I'm prompted to join the waiting list. The button [Sign up for the technical preview](https://githubnext.com/projects/copilot-voice/) doesn't do anything but redirects to https://githubnext.com/

**Reply by AliAkhgar** [reply, marked as answer]
there was an email sent from githubnext announcing that copilot voice will not be published and maintained as a product and its over. atleast for us.

**Reply by Drodarious** [reply]
This is sad to hear, as someone with an RSI and frequent pain, this gave me hope 😢
